### DISCUSSION TITLE
[QUESTION] How to send key sequene?

### DISCUSSION THREAD

**Discussion by cskollar** [taligentx/dscKeybusInterface#115] [dscKeybusInterface Discussion]
I would like to make a "night mode" option in Home Assistant that bypass 3 zones (05,07,15) and then sets a stay arm.

This key sequence works on my PK5500 keypad (access code not needed to bypass): 

*1050715#

So, I tried this in my code:

`dsc.write('*1050715#s'); `

But as I see, the panel only got the 's' command, and sent error beep because the 3 zones were open and not bypassed.

What is the correct format for a key sequence string in the dsc.write()?

Thanks a lot!

**Comment by cskollar** [comment]
Ahh, I've used `"` instead `'` in:

`dsc.write("*1050715#s");`

.. and everything is working well :)

**Comment by taligentx** [comment]
Good to hear you caught the issue, thanks for the update!
